authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -60,17 +57,38 @@ data:
 [id="sts-mode-installing"]
 == Installing an {product-title} cluster configured for manual mode with STS
 
-To install a cluster that is configured to use the CCO in manual mode with STS in {product-title} version 4.7:
+To install a cluster that is configured to use the Cloud Credential Operator (CCO) in manual mode with STS:
 
-. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-installing-manual-config_cco-mode-sts[Create the required AWS resources]
+//[pre-4.8]. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-installing-manual-config_cco-mode-sts[Create the required AWS resources]
+. xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-configuring_cco-mode-sts[Configure the Cloud Credential Operator utility]
+. Create the required AWS resources xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-creating-individually_cco-mode-sts[individually], or xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-creating-at-once_cco-mode-sts[with a single command]
 . xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-installing-manual-run-installer_cco-mode-sts[Run the {product-title} installer]
 . xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-installing-verifying_cco-mode-sts[Verify that the cluster is using short-lived credentials]
 
-//Task part 1: Creating AWS resources manually
-include::modules/sts-mode-installing-manual-config.adoc[leveloffset=+2]
+[NOTE]
+====
+Because the cluster is operating in manual mode when using STS, it is not able to create new credentials for components with the permissions that they require. When upgrading to a different minor version of {product-title}, there are often new AWS permission requirements. Before upgrading a cluster that is using STS, the cluster administrator must manually ensure that the AWS permissions are sufficient for existing components and available to any new components.
+====
+
+//[pre-4.8]Task part 1: Creating AWS resources manually
+//include::modules/sts-mode-installing-manual-config.adoc[leveloffset=+2]
+
+//Task part 1: Configuring the Cloud Credential Operator utility
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+2]
+
+[id="sts-mode-create-aws-resources-ccoctl"]
+=== Creating AWS resources with the Cloud Credential Operator utility
```

Comment:
Everything looks good on the front end but wondering why this part isn't a module? I'm sure it's for good reason but can't wrap my head around it. 😀